packages/containers/src/HomeListView/HomeListView.container.js

```diff
@@ -26,16 +26,7 @@ class HomeListView extends React.Component {
 	}
 
 	render() {
-		const { sidepanel, list, children, ...props } = this.props;
-		return (
-			<Component
-				sidepanel={sidepanel}
-				list={list}
-				{...props}
-			>
-				{children}
-			</Component>
-		);
+		return (<Component {...this.props} />);
```

Comment:
But it seems to work that way, I don't understand.
http://155.talend.surge.sh/containers/?selectedKind=HomeListViewExample&selectedStory=Default&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel